doc/dependencies.md

```diff
@@ -28,3 +28,18 @@ These are the dependencies currently used by Bitcoin Core. You can find instruct
 | xkbcommon |  |  |  |  | [Yes](https://github.com/bitcoin/bitcoin/blob/master/depends/packages/qt.mk#L86) (Linux only) |
 | ZeroMQ | [4.3.1](https://github.com/zeromq/libzmq/releases) | 4.0.0 | No |  |  |
 | zlib | [1.2.11](https://zlib.net/) |  |  |  | No |
+
+Controlling dependencies
+------------------------
+Some dependencies are not needed in all configurations. The following are some factors that affect the dependency list.
+
+#### Options passed to `./configure`
+* **miniupnpc** is not needed with  `--with-miniupnpc=no`.
```

Comment:
Nit: would be nice if the dependency names used the same capitalization as in the table above (MiniUPnPc, ZeroMQ, Berkeley DB, Qt).

Reply:
Agreed! Will fix.